cmd/root.go

```diff
@@ -254,16 +254,20 @@ Configuration using environment variables
 Localhost Admin Server
 
     The Proxy includes support for an admin server on localhost. By default,
-    the admin server is not enabled. To enable the server, pass the --debug
-    flag. This will start the server on localhost at port 9091. To change the
-    port, use the --admin-port flag.
+    the admin server is not enabled. To enable the server, pass the --debug or
+    --quitquitquit flag. This will start the server on localhost at port 9091.
+    To change the port, use the --admin-port flag.
 
-    The admin server includes Go's pprof tool and is available at
+    When --debug is passed, the admin server enables Go's profiler available at
     /debug/pprof/.
 
     See the documentation on pprof for details on how to use the
     profiler at https://pkg.go.dev/net/http/pprof.
 
+    When --quitquitquit is passed, the admin server adds an endpoint for
```

Comment:
"...the admin server exits gracefully when it receives a POST request at /quitquitquit. When..."

Reply:
That's better.